### PR TITLE
Allow skipping sending emails for some apps

### PR DESCRIPTION
**Pull request by dmathieu**
So direwolf tests don't notify.

cc @heroku/core-services  @danp

### CODE REVIEW

File: lib/mediators/messages/creator.rb [hunk 1]
```diff
@@ -16,23 +16,42 @@ def initialize(producer:, title:, body:, action_label:, action_url:, target_type
...
+        msg[:title] = msg[:title].gsub(APP, app_info.fetch("name"))
+        msg[:body] = msg[:body].gsub(APP, app_info.fetch("name"))
+      end
+      msg.save
+    end
+
+    def skip_email?(msg)
+      case msg.target_type
+      when Message::APP
+        !!app_info["name"].match(/^direwolf-/)
```
Comment (tt) [inline lib/mediators/messages/creator.rb]: So what if the main Direwolf app is called "direwolf-production"?

**Comment by tt** [comment]
I really don't think our components should do any filtering. If you're a collaborator on a test app or a member of a test org, you receive these by design. We shouldn't add conditional logic to suppress expected behavior.

**Comment by dmcinnes** [comment]
I agree with @tt, we've recently had problems with filtering of certain emails in PremiumRush, perfectly understandable from the historical perspective of how orgs were set up, but once we moved to credit card orgs we accidentally started filtering out billing emails and it took a while to track down.

**Comment by danp** [comment]
Alright, I created a filter. Feel free to close.

**Pull request closed without merging**